tests/data/v2.0/test_polymorphic_specs/swagger.json

```diff
@@ -0,0 +1,130 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Small example of polymorphic Swaggr specs",
+    "version": "0.0.0",
+    "title": "Pets"
+  },
+  "basePath": "/",
+  "paths": {
+    "/pets": {
+      "get": {
+        "summary": "Get all the pets",
+        "operationId": "get_pets",
+        "responses": {
+          "200": {
+            "description": "Pet List"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BaseObject": {
+      "allOf": [
+        {"$ref": "#/definitions/GenericPet"}
+      ]
+    },
+    "GenericPet": {
+      "type": "object",
+      "title": "GenericPet",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "weight": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "type",
+        "weight"
+      ]
+    },
```

Comment:
Could you already set the discriminator in this file? That way it's less confusing as to why it's named polymorphic. You can then change (or delete) the discriminator for your tests just like you do now.